index.html

```diff
@@ -1,10 +1,146 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="styles.css">
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Concept Opdracht</title>
 </head>
 <body>
-
+<header>
+    <nav>
+        <ul>
+            <li><a href="#studies">Studies</a></li>
+            <li><a href="#services">Services</a></li>
+            <li><a href="#contact">Contact Us</a></li>
+        </ul>
+    </nav>
+    <div class="small-inner-container">
+        <h1>Your website is an investment in the success of your business.</h1>
+        <p>We are here to help you succeed.</p>
+        <button type="button">Let's work together</button>
+    </div>
+    <img src="assets/squiggle_line_stripe.svg" alt="vertical stripe" class="squiggle-line-stripe"/>
+</header>
+<main>
+    <section id="introduction">
+        <div class="small-inner-container">
+            <h2>Rekupe is a digital agency in Los Angeles.</h2>
```

Comment:
We gebruiken `<h2>` meestal als section titels, dus gezien dit vooral een introductie-gedeelte is, zou ik hier h3 of lager voor gebruiken!

Note: je hebt het _zo_ goed gedaan dat dit de ENIGE opmerking is die ik je kan geven op je HTML, haha 🤣 Terri for president!!